WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/BackupDownloadViewModel.kt

```diff
@@ -80,6 +85,11 @@ class BackupDownloadViewModel @Inject constructor(
         }
     }
 
+    fun addSnackbarMessageSource(snackbarEvents: LiveData<Event<SnackbarMessageHolder>>) {
+        _snackbarEvents.addSource(snackbarEvents) { event ->
+            _snackbarEvents.value = event
+        }
+    }
```

Comment:
Np: Newline can be inserted after this function ends.

Reply:
Updated. Thanks